library/test/utils/url.ts

```diff
@@ -0,0 +1,3 @@
+export function getPageUrl(pageName: string, blossomId: string): string {
+  return `http://localhost:9000/test/webpages/${pageName}/?blossomId=${blossomId}`
```

Comment:
can the `localhost:9000` be a parameter (with this default) as well? - or maybe get it from some constant